server/routes/index.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+
+const router = express.Router();
+
+const loginUser = require('../controllers/loginUser');
```

Comment:
you can import both two lines in one line if you make export for both in index controller file (see what I did at the last PR)